src/commands/emasser/post/poams.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable valid-jsdoc */
```

Comment:
Just curious - why does this file have this line?

Reply:
Because there are comments without a return statement.